app/src/routes/+page.server.ts

```diff
@@ -1,16 +1,20 @@
-import { client } from "$lib/sanity/client";
-import { eventsQuery as query } from "$lib/sanity/queries";
 import type { PageServerLoad } from "./$types";
 import type { Event } from "$models/sanity.types";
+import { serverClientWithoutStega } from "$lib/server/sanity/client";
+import { futureEventsQuery, pastEventsQuery } from "$lib/server/sanity/queries";
 
 export const load: PageServerLoad = async (event) => {
   const url = new URL(event.url);
   const category = url.searchParams.get("category");
 
-  const events: Event[] = await client.fetch(query, { category });
+  const futureEvents: Event[] = await serverClientWithoutStega.fetch(futureEventsQuery, {
```

Comment:
Det kan være at client er bra? Da får man live mode fra sanity tror jeg med stega.

Reply:
Aa oki, trodde du sa noe om at vi skulle ha uten 😅

Reply:
Ja bør ta av stega dersom man sender mail eller lagrer til database! Men for å vise innhold i html så er det good 👍

Reply:
Ohhh det var sånn det var ja